tensorflow/core/BUILD

```diff
@@ -1216,6 +1216,9 @@ LIB_INTERNAL_WINDOWS_DEPS = glob(
         "platform/*.cc",
         "platform/profile_utils/**/*.h",
         "platform/profile_utils/**/*.cc",
+    ] + [
```

Comment:
Do we need this line?
Maybe we can have a comment to explain these, but not sure about two separate lists.